src/footer/footer.macros.html

```diff
@@ -0,0 +1,59 @@
+{#
+  * @overview Macros for standard and simplified site Footers
+#}
+
+{% from '../icons/icons.macros.html' import icon as icon %}
+
+{#
+  * Renders a standard or simplified footer
+#}
+{% macro footer(simple=false, class=null, id=null) %}
```

Comment:
I'd rather use `true`, `false`, and `null` consistently. What do you think of standardizing on `true` and `false`?